.github/workflows/ci.yml

```diff
@@ -52,9 +52,9 @@ jobs:
         run: |
           sudo apt-get update && DEBIAN_FRONTEND=noninteractive sudo apt-get install -y postgresql
           sudo systemctl start postgresql
-          sudo -u postgres createuser cratesfyi -w
-          sudo -u postgres createdb cratesfyi -O cratesfyi
-          echo "::set-env name=CRATESFYI_DATABASE_URL::postgresql://cratesfyi@%2Fvar%2Frun%2Fpostgresql/cratesfyi"
+          sudo -u postgres createuser $(whoami) -w
+          sudo -u postgres createdb cratesfyi -O $(whoami)
+          echo "::set-env name=CRATESFYI_DATABASE_URL::postgresql://$(whoami)@%2Fvar%2Frun%2Fpostgresql/cratesfyi"
           sudo -u postgres psql -d cratesfyi -c "CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;"
```

Comment:
```suggestion
          sudo -u postgres psql -d $(whoami) -c "CREATE EXTENSION IF NOT EXISTS fuzzystrmatch;"
```

Reply:
What what why did the test pass D:

Reply:
The database name doesn't have to be the user's name

Reply:
The name of the database was changed to `cratesfyi` to make it a bit more consistent with the docker setup, but the user used to login was left as `$(whoami)` because it's using the local machine's user accounts to login.